### ISSUE TITLE
Vertical scrollbar can't be clicked and dragged in exported property dropdown selection

### ISSUE THREAD

**Issue by enxas** [godotengine/godot#78760]
### Godot version

v4.1.beta3.mono.official [ada712e06]

### System information

Godot v4.1.beta3.mono - Windows 10.0.19045 - Vulkan (Forward+) - dedicated NVIDIA GeForce GTX 1660 Ti with Max-Q Design (NVIDIA; 31.0.15.3179) - Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 Threads)

### Issue description

Vertical scrollbar can't be clicked and dragged in exported property dropdown selection. When you hover over a scrollbar it will light up, but can't be clicked and dragged like other scrollbars.

### Steps to reproduce

Export a property that has a lot of options to select like `[Export] Resource resource;`
In the inspector open property selection dropdown then try to click and drag vertical scrollbar.

https://github.com/godotengine/godot/assets/8630984/60d8f85b-c72a-46b9-bcf7-1f28e680111a

### Minimal reproduction project

NA

**Comment by enxas** [comment]
Tested with Godot v4.1.rc.mono.custom_build [3c352a5a7] and same problem still exist

**Comment by FlooferLand** [comment]
This problem has started happening for me in 4.0 as well, not 4.1 related but I really hope it gets fixed in 4.1

It happens for me when trying to pick the type for a resource, and also on the "Add Metadata Property" dropdown box. There might be other cases too

**Comment by Maran23** [comment]
This is a very weird issue. I debugged this a bit and found out the following:
For `PopupMenu`, no left mouse click event is sent. I first thought this may be filtered out by Godot, but I could track this down to the low level Windows code.

Other classes like `Dialog` or even `PopupPanel` are not affected.
Also, this only happens when embedded subviewports is not enabled, it can then also reproduced in your own games.

That brings me to multiple questions:
- Does this also happens on other platforms? All reports come from Windows, which probably makes sense since I did not get a `WM_LBUTTONDOWN` for popup menus.
- Why is only `PopupMenu` affected? Why not `Popup`, `PopupPanel`?
